lib/pyfrc/sim/ui.py

```diff
@@ -335,21 +335,35 @@ def _set_mode():
             pass
         else:
             auton = tk.LabelFrame(ctrl_frame, text='Autonomous')
-            
+
             self.autobox = Combobox(auton, state='readonly')
             self.autobox.bind('<<ComboboxSelected>>', self.on_auton_selected)
             self.autobox['width'] = 12
             self.autobox.pack(fill=tk.X)
-            
+
             Tooltip.create(self.autobox, "Use robotpy_ext.autonomous.AutonomousModeSelector to use this selection box")
-            
+
             from networktables.util import ChooserControl
             self.auton_ctrl = ChooserControl('Autonomous Mode',
                                               lambda v: self.idle_add(self.on_auton_choices, v),
                                               lambda v: self.idle_add(self.on_auton_selection, v))
-            
+
             auton.pack(side=tk.TOP)
-        
+
+            gamedata = tk.LabelFrame(ctrl_frame, text='Game Data')
+
+            self.gamedatabox = Combobox(gamedata, state='readonly')
```

Comment:
readonly?

Reply:
Took a read of the tkinter.ttk docs. I think we want to future-proof this and let the user type arbitrary data (so keep the default state). Would rather not have users wait for us to update this next year if this is being used in future games.

Reply:
Agreed

Reply:
I think if you just get rid of the readonly thing now then we don't have to worry about it next year.